runelite-client/src/main/java/net/runelite/client/plugins/agility/AgilityPlugin.java

```diff
@@ -38,33 +38,18 @@
 import lombok.Getter;
 import lombok.Setter;
 import lombok.extern.slf4j.Slf4j;
-import net.runelite.api.Client;
```

Comment:
Revert all of these import changes. Your static imports and normal imports should be configured as described in the wiki.

https://github.com/runelite/runelite/wiki/Code-Conventions

Reply:
Thanks for the review, I applied those settings in the code conventions and re-committed that file